src/ControlKnob.vue

```diff
@@ -210,6 +221,7 @@ function changeValue(change: number) {
 }
 
 function keyDownListener(event: KeyboardEvent) {
+  //Update the shift modifier here already, otherwise the precise mode is not triggered properly
```

Comment:
This is a good comment to have but is formatted incorrectly (missing a space at the end). Eslint should have catched this -- are you sure your editor is configured properly?

Reply:
Done.